ci-operator/step-registry/vsphere/remote-worker/vsphere-remote-worker-commands.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+infrastructure_id="$(oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster)"
+template="$(oc get machineset -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.template}' -n openshift-machine-api)"
+datacenter="$(oc get machineset -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.workspace.datacenter}' -n openshift-machine-api)"
+datastore="$(oc get machineset -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.workspace.datastore}' -n openshift-machine-api)"
+folder="$(oc get machineset -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.workspace.folder}' -n openshift-machine-api)"
+resourcePool="$(oc get machineset -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.workspace.resourcePool}' -n openshift-machine-api)"
+server="$(oc get machineset -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.workspace.server}' -n openshift-machine-api)"
+
+remote_worker_number="${REMOTEWORKER_NUMBER}"
+
+cat >>"${SHARED_DIR}/remote-worker-machineset.yaml" <<EOF
```

Comment:
Can we dump out the existing machineset from the cluster, then modify it to meet our requirement, maybe we need to take machineset compatibility between versions.

Reply:
Here is a good example for similar case - https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/inframachineset/ipi-conf-inframachineset-commands.sh#L45-L57

Reply:
ACK, will address it.